app/views/users/show.html.erb

```diff
@@ -1,6 +1,44 @@
+<%= render 'layouts/navigation', current_user: @current_user%>
+
 <p><%= flash[:notice] %></p>
-<h1>Hey, <%= @user[:name] %>!</h1>
+<p><%= flash[:alert] %></p>
+<h1>Hey, <%= @current_user[:name] %>!</h1>
+
+<table>
+    <thead>
+        <strong>EVENTS YOU CREATED</strong>
+    </thead>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <th>Location</th>
+            <th colspan="3">Date</th>
+        </tr>
+
+        <% @user_events.each do |event| %>
+            <tr>
+                <td><%= event.description %></td>
+                <td><%= event.location %></td>
+                <td><%= event.date %></td>
+            </tr>
+        <% end %>
+    </tbody>
+</table>
+
+<hr>
+
+<h5>UPCOMING EVENTS</h5>
+
+<ul>
+    <% @user_attending_events.where('date >= ?', Date.current).each do |event| %>
```

Comment:
- [ ] Use the past and future event scopes at the user controller instead of writing the whole ActiveRecord query directly in the view. The view is only for the UI of the app, logic like this should be at the model level.